src/database/sql_database.cc

```diff
@@ -1724,11 +1724,9 @@ long long SQLDatabase::getFileStats(const StatsParam& stats)
     }
     }
     auto query = fmt::format("SELECT {} FROM {}{} WHERE {}", mode, searchColumnMapper->tableQuoted(), join, fmt::join(where, " AND "));
-    auto res = select(query);
-    std::unique_ptr<SQLRow> row;
-    if (res && (row = res->nextRow())) {
-        return row->col_long(0, 0);
-    }
+    if (auto res = select(query))
```

Comment:
Using assignments in conditions is no good code style: Each statement should perform one operation.